### ISSUE TITLE
Support non-hard-coded dictionary of words

### ISSUE THREAD

**Issue by l2lam** [l2lam/Word-guessing-game-demo#8]
The game would be far friendly for administrators if they could paste, upload or point the game at a dictionary of words to use in the game rather than updating the program code to set their own dictionary.

**Comment by l2lam** [comment]
Yeah, but consider who the target audience is it might be... How techie did we expect them to be?  A science teacher might want to use this game as a fun quiz.  A grade one teacher might want to use it for math practice, etc.  Good design would consider ways to make this process as easy, or even enjoyable, as possible.  This would be a good thing to prototype and test on people.

**Comment by WhyNot180** [comment]
hmm, perhaps something that could be changed in a menu option then

**Comment by l2lam** [comment]
Some other options to consider:
- CSV file upload because a lot of people know how to use spreadsheets
- input box where they can type it paste some delineated list of words/phrases
- choose from last based on an online resource, searchable by category/grades, etc
- present a dictionary editor to manage the dictionary

**Comment by UserC2** [comment]
> Some other options to consider:
> 
> * CSV file upload because a lot of people know how to use spreadsheets
You may want to include instructions on how to export a CSV file and use it with the program (or link to a help page for a few popular spreadsheet programs).

**Comment by WhyNot180** [comment]
The main bulk of the code has been merged with main. Next step is to implement a selector of importers.
